docs/area-owners.md

```diff
@@ -2,22 +2,22 @@ The below table lists all the `area-`labels used in the `aspnetcore` repository
 
 | Area label | Owner | Description|
 |--- | ---| --- |
-| area-azure | anurse | |
+| area-azure | BrennanConroy | |
 | area-blazor | mkArtakMSFT | Blazor server and Blazor WASM related |
-| area-commandlinetools | anurse, mkArtakMSFT | dev certs, dotnet watch,  |
-| area-dataprotection | anurse | |
+| area-commandlinetools |  mkArtakMSFT | dev certs, dotnet watch,  |
+| area-dataprotection | BrennanConroy | |
```

Comment:
@blowdart Owns this one I think

Reply:
If this is engineering ownership, no I don't. 

@Pilchie has yet to assign anyone, so I'd assign it to him :)

Reply:
```suggestion
| area-dataprotection | Pilchie | |
```